tests/unittest/test_model_weight_share.py

```diff
@@ -28,6 +28,7 @@ def _check_initialized(net):
             return False
         return True
 
+@pytest.mark.filterwarnings('ignore:UserWarning')
```

Comment:
Does this work?
Previously we use the following https://github.com/dmlc/gluon-nlp/blob/e0323089d98c9e92008c73f155161da24c41d96d/scripts/tests/test_dataprocessor.py#L72-L74

Either is fine

Reply:
It seems `pytest.mark.filterwarnings` does not work here. I have updated file to use this version.